web/cobrands/sass/_layout.scss

```diff
@@ -272,6 +273,7 @@ body.mappage {
       left: 0;
       right: 0;
       background: $map_nav_bg;
+      height: $mappage-header-height;
```

Comment:
I think this might cause the navigation to break if it wraps onto two lines, e.g. http://www.fiksgatami.no/report/36972 at an appropriate width. `nav-wrapper-2` in this file further up has a `min-height` of 4em which I guess should be this variable, rather than adding this line here, does that work?